task/sbom-json-check/0.1/sbom-json-check.yaml

```diff
@@ -38,6 +38,8 @@ spec:
 
       if ! oc image extract "${image_with_digest}" --path '/root/buildinfo/content_manifests/*:/manifests/'; then
         echo "Failed to extract manifests from image ${image_with_digest}"
+        note="Task $(context.task.name) failed: Failed to extract manifests from image ${image_with_digest} with `oc extract`. For detailed error, check the Tekton task log."
```

Comment:
SR: note="Task $(context.task.name) failed: Failed to extract manifests from image ${image_with_digest} with `oc extract`. For details, check the Tekton task log."

- for some reason, this is rendering on the Preview tab in weird italics. please ignore that!

Reply:
Fixed.